packages/kit/src/runtime/client/client.js

```diff
@@ -1947,4 +1953,12 @@ if (DEV) {
 		}
 		console_warn(...args);
 	};
+
+	if (import.meta.hot) {
+		import.meta.hot.on('vite:beforeUpdate', () => {
+			if (errored) {
+				location.reload();
```

Comment:
should we add something here like `console.warn('Hit an unrecoverable error. Reloading the page')`?

Reply:
no-one would see the message, because the page would reload the moment it was logged. we could add it when `errored` is set to `true` though

Reply:
I think by default that's true, but I believe there's a checkbox you can toggle to persist it across a page navigation and I thought people might try that if they were confused why the reload was happening